src/libs/vue-h-zoom.js

```diff
@@ -110,7 +121,20 @@ export default {
       return posY
     },
     zoomStyle: function () {
-      return {
+      return this.containImage ? {
```

Comment:
can we put all similar props in one object first, then replace the different once in the next lines?